docs/www/autotune.md

```diff
@@ -6,7 +6,7 @@ order: 1
 # Designed for performance
 
 Great news! Redpanda comes with an autotuner that detects the optimal settings for your hardware.
-To get the best performance for your hardware, set Redpanda to [production mode](./docs/production-deployment).
+To get the best performance for your hardware, set Redpanda to [production mode](https://vectorized.io/docs/production-deployment).
```

Comment:
Is this fixing an issue on the docs site? It looks to me like it should not be tied to a published domain because local renderings that depend on relative paths would not work as expected.

If it is needed, it should be in a separate commit that explains the issue. Currently it is contained in the same commit that addresses a memory reservation limit.

Reply:
@dotnwat I don't know why this file is even changing. I didn't touch it. Let me see why this is happening.

Reply:
Done.